File: Python/foobar.py
def output_val(number):
    if number % 15 == 0:
        return "foobar"
    elif i % 5 == 0:
        return "bar"
    elif i % 3 == 0:
        return "foo"
    else:
        return number


for i in range(1, 101):
    if i != 100:
        delimiter = ', '
        ending = ''
    else:
        delimiter = ''
        ending = '\n'
    print(output_val(i), delimiter, sep='', end=ending)

File: foobar.php
<?php

for ($i=1; $i <= 100; $i++) {
  if ($i % 15 === 0) {
    echo "foobar";
  }
  elseif ($i % 5 === 0) {
    echo "bar";
  }
  elseif ($i % 3 === 0) {
    echo "foo";
  }
  else echo $i;

  echo ($i === 100) ? "\n" :", ";
}

?>
